### PR TITLE
Milestone3 contact form

### PR DESCRIPTION
**Pull request by estebanmual**
### Home Contact form. 

In this milestone we
- Added the contact form.
- Added the contact information and the inputs
- Linked the contact form with Formspree
- Added the footer icons.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,169 +1,343 @@
...
+              class="icon-angelist"
+              src="/03_Resources/01_Graphics/02_Stock/ic_angellist.svg"
+              alt="Angelist icon"
+          /></a>
+          <a href="#"
+            ><img
+              class="icon-medium"
+              src="/03_Resources/01_Graphics/02_Stock/ic_medium.svg"
+              alt="Medium icon"
+          /></a>
```
Comment (rotimiazeez) [inline index.html]: - [x] Kindly make sure all the image links provided are not broken to match the Figma design😎
- Your design 👇
screenshot (30)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi Team👋, 
## Good job so far!
### Summary 👏
- No linter errors ✔
- Good commit message 👍
- Descriptive PR message😊
- Good Readme 🚀
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Fix the image for the entire page for a great UI(User Interface)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -1,169 +1,343 @@
...
+              class="icon-angelist"
+              src="03_Resources/01_Graphics/02_Stock/ic_angellist.svg"
+              alt="Angelist icon"
+          /></a>
+          <a href="#"
+            ><img
+              class="icon-medium"
+              src="03_Resources/01_Graphics/02_Stock/ic_medium.svg"
+              alt="Medium icon"
+          /></a>
```
Comment (mateo951) [inline index.html:338]: Addressed 🔥😎

image

**Review by mateo951 — APPROVED** [review]
Hi @Estebanmual and @mengiefen,

Both of you have done a great job with the project 🧐 Splendid work at implementing the changes required by your previous reviewer 🔥 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (estebanmual) [reply index.html:338]: Awesome!!!!! 🔥🔥🔥🔥🔥🔥

**Pull request merged**